apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/images-02.php

```diff
@@ -0,0 +1,99 @@
+
+<!-- wp:paragraph {"align":"right","className":"margin-bottom-half","style":{"typography":{"lineHeight":"1.2"}}} -->
+<p class="has-text-align-right margin-bottom-half" style="line-height:1.2"><strong>%7$s</strong><br><em>%8$s</em></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:image {"id":1277,"sizeSlug":"full","className":"margin-top-half"} -->
+<figure class="wp-block-image size-full margin-top-half"><a href="#"><img src="https://dotcompatterns.files.wordpress.com/2020/05/jean-philippe-delberghe-exbd9gxafhs-unsplash.jpg" alt="" class="wp-image-1277"/></a></figure>
+<!-- /wp:image --></div>
+<!-- /wp:jetpack/layout-grid-column --></div>
+<!-- /wp:jetpack/layout-grid -->
+
+<!-- wp:spacer {"height":60} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:jetpack/layout-grid {"gutterSize":"huge","column1DesktopSpan":6,"column1DesktopOffset":1,"column1TabletSpan":8,"column1MobileSpan":4,"column2DesktopOffset":6,"className":"column1-desktop-grid__span-6 column1-desktop-grid__start-2 column1-desktop-grid__row-1 column1-tablet-grid__span-8 column1-tablet-grid__row-1 column1-mobile-grid__span-4 column1-mobile-grid__row-1"} -->
+<div class="wp-block-jetpack-layout-grid alignfull column1-desktop-grid__span-6 column1-desktop-grid__start-2 column1-desktop-grid__row-1 column1-tablet-grid__span-8 column1-tablet-grid__row-1 column1-mobile-grid__span-4 column1-mobile-grid__row-1 wp-block-jetpack-layout-gutter__huge"><!-- wp:jetpack/layout-grid-column -->
+<div class="wp-block-jetpack-layout-grid-column wp-block-jetpack-layout-grid__padding-none"><!-- wp:paragraph {"align":"right","className":"margin-bottom-half","style":{"typography":{"lineHeight":"1.2"}}} -->
+<p class="has-text-align-right margin-bottom-half" style="line-height:1.2"><strong>%9$s</strong><br><em>%10$s</em> </p>
+<!-- /wp:paragraph -->
+
+<!-- wp:image {"id":1325,"sizeSlug":"large","className":"margin-top-half"} -->
+<figure class="wp-block-image size-large margin-top-half"><a href="#"><img src="https://dotcompatterns.files.wordpress.com/2020/06/sofia-kuniakina-rt990o6gexk-unsplash.jpg?w=1024" alt="" class="wp-image-1325"/></a></figure>
+<!-- /wp:image --></div>
+<!-- /wp:jetpack/layout-grid-column --></div>
+<!-- /wp:jetpack/layout-grid -->
+';
+
+return array(
+	'__file'     => 'wp_block',
+	'title'      => esc_html__( 'Images', 'full-site-editing' ),
+	'categories' => array( 'gallery', 'images' ),
+	'content'    => sprintf(
+		$markup,
+		esc_html__( 'Summer', 'full-site-editing' ),
+		esc_html__( '2019', 'full-site-editing' ),
+		esc_html__( 'Failed Encounter', 'full-site-editing' ),
+		esc_html__( '2019', 'full-site-editing' ),
+		esc_html__( 'Movement', 'full-site-editing' ),
+		esc_html__( '2018', 'full-site-editing' ),
+		esc_html__( 'Retratos', 'full-site-editing' ),
+		esc_html__( '2017', 'full-site-editing' ),
+		esc_html__( 'Disappearance', 'full-site-editing' ),
+		esc_html__( '2016', 'full-site-editing' ),
```

Comment:
This is throwing a PHPCS error (trailing comma in a function call)

Reply:
Derp. I'll fix it. :)